python/metatensor-learn/metatensor/learn/nn/module_map.py

```diff
@@ -341,3 +342,51 @@ def repr_as_module_dict(self) -> str:
             representation += f"  ({key!r}): {self[i]!r}\n"
         representation += ")"
         return representation
+
+    def to(self, *args, **kwargs):
```

Comment:
this seems extremely overkill. Why is this not using `super().to()`?

```py
def to(...):
    self._in_keys = ...
    self._out_properties = ...
    
    super().to(...)
```

Reply:
Ok! But we'll need one more line to extract the device